.gitignore

```diff
@@ -84,3 +84,4 @@ crashlytics-build.properties
 .attach_pid*
 .vscode/
 .java-version
+*Dockerfile
```

Comment:
I am on Mac M1, and buildpacks can not (currently) build images for that, there is on open defect. So the only way for me to test things locally is to build the image from a Dockerfile, thus this ignore here. Hope it's OK.